pandas/tests/extension/decimal/test_decimal.py

```diff
@@ -174,7 +177,8 @@ class TestCasting(BaseDecimal, base.BaseCastingTests):
 
 
 class TestGroupby(BaseDecimal, base.BaseGroupbyTests):
-    pass
+    pytestmark = pytest.mark.skipif(compat.PY2,
```

Comment:
dont' you have a ``@unhashable`` decorator for this?

Reply:
Not sure if we can use it. I've only used that as a regular `skip`, not as a pytestmark or as a `skipif`